core/src/main/java/com/google/errorprone/refaster/RefasterScanner.java

```diff
@@ -101,7 +101,7 @@ public Void visitVariable(VariableTree node, Context context) {
 
   @Override
   public Void scan(Tree tree, Context context) {
-    if (tree == null) {
+    if (tree == null || tree instanceof JCTree.JCPackageDecl) {
```

Comment:
Ideally we'd do this:
```suggestion
    if (tree == null || tree instanceof PackageTree) {
```
But `PackageTree` is marked `@since 9`. Perhaps we should wait for (or contribute) the changes necessary to target JDK 11, so that we can make this nice. See https://github.com/google/error-prone/issues/2783#issuecomment-1000645917.

_Edit_: it's just IDEA flagging this; the build does pass with this change. In light of the above I think it's fine to go ahead with this then.